test/test_other.py

```diff
@@ -11949,50 +11898,7 @@ def print_percent(actual, expected):
       self.run_process(terser + ['-b', 'beautify=true', 'a.js', '-o', 'pretty.js'], env=shared.env_with_node_in_path())
       self.assertFileContents(js_out, read_file('pretty.js'))
 
-    obtained_results = {}
-    total_output_size = 0
-    total_expected_size = 0
-    total_output_size_gz = 0
-    total_expected_size_gz = 0
-    for f in outputs:
-      f_gz = f + '.gz'
-      expected_size = expected_results[f] if f in expected_results else float('inf')
-      expected_size_gz = expected_results[f_gz] if f_gz in expected_results else float('inf')
-      size, size_gz = get_file_and_gz_sizes(f)
-
-      obtained_results[f] = size
-      obtained_results[f_gz] = size_gz
-
-      if size != expected_size and (f.endswith(('.js', '.html'))):
-        print('Contents of ' + f + ': ')
-        print(read_file(f))
-
-      print('size of ' + f + ' == ' + str(size) + ', expected ' + str(expected_size) + ', delta=' + str(size - expected_size) + print_percent(size, expected_size))
-      print('size of ' + f_gz + ' == ' + str(size_gz) + ', expected ' + str(expected_size_gz) + ', delta=' + str(size_gz - expected_size_gz) + print_percent(size_gz, expected_size_gz))
-
-      # N.B. even though the test code above prints out gzip compressed sizes, regression testing is done against uncompressed sizes
-      # this is because optimizing for compressed sizes can be unpredictable and sometimes counterproductive
-      total_output_size += size
-      total_expected_size += expected_size
-
-      total_output_size_gz += size_gz
-      total_expected_size_gz += expected_size_gz
-
-    obtained_results['total'] = total_output_size
-    obtained_results['total_gz'] = total_output_size_gz
-    print('Total output size=' + str(total_output_size) + ' bytes, expected total size=' + str(total_expected_size) + ', delta=' + str(total_output_size - total_expected_size) + print_percent(total_output_size, total_expected_size))
-    print('Total output size gzipped=' + str(total_output_size_gz) + ' bytes, expected total size gzipped=' + str(total_expected_size_gz) + ', delta=' + str(total_output_size_gz - total_expected_size_gz) + print_percent(total_output_size_gz, total_expected_size_gz))
```

Comment:
This output delta information expresses and a percent might still be useful, did we loose it?

Reply:
Nope, it's still there, just in the common.py.